lib/challenge.js

```diff
@@ -20,6 +20,17 @@
 
 // Answer Starts Here
 
+//'User'
+//Description our calendar manages tasks
+//Every date should have
+//- start date and time
+//- duration
+//- end date and time
+//Every task will have
+// - task description
+// - duration
+// - recurrence
+
```

Comment:
You broke the format here, describing the properties of dates and tasks, whereas the question would have been satisfied with you dropping one of those, but asked for a list of calendar behaviors, as well. Half credit.